lib/jsdom/living/nodes/HTMLTextAreaElement-impl.js

```diff
@@ -69,12 +71,14 @@ class HTMLTextAreaElementImpl extends HTMLElementImpl {
   }
 
   set value(val) {
+    const oldValue = this._getAPIValue();
```

Comment:
nit: the spec calls it _oldAPIValue_.

Reply:
Thanks, I renamed this to `oldAPIValue` and added a comment linking to the spec.